hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/DeleteOperation.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.multimap.impl.operations;
+
+import com.hazelcast.config.MultiMapConfig;
+import com.hazelcast.core.EntryEventType;
+import com.hazelcast.multimap.impl.MultiMapContainer;
+import com.hazelcast.multimap.impl.MultiMapDataSerializerHook;
+import com.hazelcast.multimap.impl.MultiMapRecord;
+import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.spi.Operation;
+
+import java.util.Collection;
+
+//TODO: should this not be backup aware? check other Map/remove/Delete
```

Comment:
I don't understand this TODO, the `MultiMapBackupAwareOperation` implements `BackupAwareOperation`

Reply:
Oops, embarrassing note to myself that made it in when I was just playing around and exploring the guts of Hazelcast. :)